boards/nxp/s32z2xxdc2/doc/index.rst

```diff
@@ -163,6 +165,16 @@ ADC is provided through ADC SAR controller with 2 instances. Each ADC SAR instan
 .. note::
    All channels of an instance only run on 1 group channel at the same time.
 
+EDMA
+====
+
+The EDMA modules are categorized into one EDMA4 instance (instance 3) and four EDMA3 instances
```

Comment:
The PR description said this supports eDMA3, why you mention eDMA4 here

Reply:
I removed eDMA4